openmdao/docs/features/building_blocks/components/external_code_implicit_comp.rst

```diff
@@ -0,0 +1,40 @@
+.. index:: ExternalCodeImplicitComp Example
+
+.. _externalcodeimplicitcomp_feature:
+
+************************
+ExternalCodeImplicitComp
+************************
+
+`ExternalCodeImplicitComp` is very similar to `ExternalCodeComp` in that it runs an external program in a subprocess on your
+operating system. But it treats the `Component` as an `ImplicitComponent` rather than an `ExplicitComponent`. See
+:ref:`ExternalCodeComp <externalcodecomp_feature>` for basic information about how `ExternalCodeComp` works.
+
+`ExternalCodeImplicitComp` has the same options as `ExternalCodeComp`.
+
+ExternalCodeImplicitComp Example
+--------------------------------
+
+The only difference between `ExternalCodeImplicitComp` and `ExternalCodeComp` is that for the former, a `compute`
+method must be defined, while for the latter, a `apply_nonlinear` method must be defined.
```

Comment:
*an* `apply_nonlinear` method  (or should it be *the* `compute` method and *the* `apply_linear` method... judges?)

Reply:
both ? Actually, I got the order flipped. compute for explicit and apply_nonlinear to implicit.